docs/Roadmap.md

```diff
@@ -36,35 +25,75 @@ instance. Care must be taken to keep the standalone version usable.
+grateful. Funds from these source help support the cost of running the servers, save and plan for the future, and
+incentivize the core team. Previously our goal was to remain "ad-free", that has been relaxed slightly to allow up to
+three company sponsor logos visible at the top right of the screen.
 
 ## Goals
 
-### 2021 goals
-
-- **Login support**. Support logging in to the site with GitHub, Google, etc. We will _never_ force you to log in for
-  basic features, and of course will update the Privacy Policy. I won't be selling anything to do with user info etc,
-  either: Logging in will be purely used to make _your_ life easier and allow you to manage things like shared settings
-  and configuration, listing short URLs you've created (and potentially being able to remove them); and _maybe_ being
-  able to make user-named short URLs (e.g., "godbolt.org/u/mattgodbolt/ctad-example"). This goal is a personal pet
-  project of [Matt's](http://github.com/mattgodbolt/).
-- **Multi-file support**. Multiple file compilation units to open the door to seeing LTO and maybe modules. This may
-  include being able to use a `CMake` file to build things.
-- **Modernising the codebase**. Moving the codebase to TypeScript, or something similar that will allow us to worry less
-  about differences between front-end and back-end code (old JavaScript versions), and help us attract more people to
-  the project.
-- **More compilers and libraries**. Plus finishing off the last stragglers of installation.
+### 2023 goals
+
+- **Modernising the codebase**. 2023 has to be the year of all Typescript. We need to continue pushing towards this goal
+  as we are currently blocked using old versions of some libraries due to JS/TS/node.js/ES6 modules conflicts. We're
+  still using an outdated and unsupported module loader and in order to move off it we need to update to a newer
+  loader...that itself doesn't seemingly doesn't support a mixture of typescript and javascript like we currently have.
+  In any case, moving to Typescript has improved our code quality and continues to find bugs, so we should absolutely
+  port to it for this reason alone.
+- **Improved compiler installation directory handling**. This project started in 2022 but left to rot a little, but our
+  existing approach to sharing the 1700+ compilers and libraries between running instances is not scaling well. Behind
+  the scenes we use NFS, and then for performance mount some compilers as read-only squashfs images from NFS. That
+  worked well until we had 1000+ squashfs images to mount, one per compiler. A project to administrate and update a more
+  layered squashfs image containing large numbers of unchanged compilers (think `docker` image layers) was started in
+  2022, but fell behind due to so many other issues, and Matt being pretty rubbish at prioritising things. As nodes are
+  now taking an awful long time to start up, this is becoming a problem.
```

Comment:
This is a big prio for me, long startup makes everything about deployment and testing more painful

Reply:
100% for sure. Thanks for the feedback.